synapse/lib/layer.py

```diff
@@ -1129,6 +1133,72 @@ async def waitForHot(self):
         '''
         await self.layrslab.lockdoneevent.wait()
 
+    async def _layrV2toV3(self):
+
+        bybuid = self.layrslab.initdb('bybuid')
+        sode = collections.defaultdict(dict)
+
+        tostor = []
+        lastbuid = None
+
+        logger.warning(f'Converting layer from v2 to v3 storage: {self.dirn}')
```

Comment:
we really could do an estimated magnitude based on form counts (but not an absolute count)